src/backend/distributed/master/master_node_protocol.c

```diff
@@ -556,6 +556,16 @@ GetTableDDLEvents(Oid relationId)
 	ScanKeyData scanKey[1];
 	int scanKeyCount = 1;
 	HeapTuple heapTuple = NULL;
+	OverrideSearchPath *overridePath = GetOverrideSearchPath(CurrentMemoryContext);
+
+	/*
+	 * Set search_path to NIL so that all objects outside of pg_catalog will be
+	 * schema-prefixed. pg_catalog will be added automatically when we call
+	 * PushOverrideSearchPath(), since we set addCatalog to true;
+	 */
+	overridePath->schemas = NIL;
```

Comment:
I think it looks better if we keep initialization together.

// comments
OverrideSearchPath *overridePath = GetOverrideSearchPath(CurrentMemoryContext);
overridePath->schemas = NIL;
...

or

// comments
overridePath = GetOverrideSearchPath(CurrentMemoryContext);
overridePath->schemas = NIL; 
....


Similarly for other files.